kview/src/main/resources/dev/ikm/komet/kview/controls/public-id-control.css

```diff
@@ -20,8 +19,11 @@
 }
 
 .public-id > .title-label {
-    -fx-font-family: "Noto Sans Bold";
-    -fx-font-size: inherit;
+   -fx-font-family: "Noto Sans Medium";
+   -fx-font-weight: 500;
+   -fx-font-size: 12px;
+   -fx-text-fill: black;
+   -fx-font-weight: bold;
```

Comment:
I think that unfortunately this properties in javafx css might not actually work. 
Set the font title instead of using the property. 
Like if you want a bold text use:
-fx-font-family: "Noto Sans Bold";

If you want normal text use:
-fx-font-family: "Noto Sans";

Semi bold (which is a step before bold):
-fx-font-family: "Noto Sans Semibold";

The font files must exist though...